test.config.js

```diff
@@ -1,34 +1,50 @@
-require('@babel/register')({
-  extensions: ['.ts', '.js', '.tsx', '.jsx'],
+require("@babel/register")({
+  extensions: [".ts", ".js", ".tsx", ".jsx"],
 })
 
-require('coffeescript/register')
-require('@babel/polyfill')
-require('raf/polyfill')
-require('should')
-require('./src/lib/jade_hook')
+require("coffeescript/register")
+require("@babel/polyfill")
+require("raf/polyfill")
```

Comment:
This file is shared between mocha and jest -- and all new tests are written in Jest. The import in `mocha.sh` is meant to shim jsdom within a mocha context; in jest this is done internally.